src/Mjolnir.ts

```diff
@@ -604,7 +615,7 @@ export class Mjolnir {
     }
 
     public async buildWatchedBanLists() {
-        const banLists: BanList[] = [];
+        this.banLists = [];
```

Comment:
That changes the semantics.
The original version created the banlists as local variables and only initialized `this.banLists` at the end.

Can this cause any kind of race condition?